src/index.html

```diff
@@ -11,7 +11,162 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <div class="container">
+      <form class="form"
+      action="https://github.com/mate-academy/layout_html-form/blob/master/checklist.md"
+      method="post"
+      >
+        <fieldset class="form__personal">
+          <legend>Personal information</legend>
+          <div class="form__personal-surname">
+            Surname:
+            <input
+              name="surname"
+              type="text"
+              autocomplete="off"
+            >
+          </div>
+          <div class="form__personal-name">
+            Name:
```

Comment:
add labels to each input. it will expand clickable place and make the text available to click

Reply:
checklist
<img width="888" alt="image" src="https://user-images.githubusercontent.com/102411130/221780226-5117db9e-1242-45d8-922b-112b3cebaad5.png">